account_financial_risk_ext/views/res_partner_view.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="res_partner_view_risk" model="ir.ui.view">
+        <field name="name">res.partner.view.risk</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="account_financial_risk.res_partner_view_risk" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='credit_limit']" position="after">
+                <field name="credit_balance" widget="monetary" />
```

Comment:
We should add `attrs="{'readonly': [('risk_allow_edit', '=', False)]}"`, just like other fields in the view
https://github.com/qrtl/asx-custom/blob/2dee897ace548b4df00641c0aee5d94372bf0dca/account_financial_risk/views/res_partner_view.xml#L53-L57

Reply:
@TimLai125 It's a non-stored computed field and, as such, is always readonly.